src/helm/benchmark/run_expander.py

```diff
@@ -768,7 +768,8 @@ def __init__(self, value):
     def expand(self, run_spec: RunSpec) -> List[RunSpec]:
         # Find right tokenizer given model.
         if isinstance(self.all_values, dict):
-            self.values = self.all_values[run_spec.adapter_spec.model]
+            model: str = run_spec.adapter_spec.model
+            self.values = self.all_values[model] if model in self.all_values else []
```

Comment:
Isn't this going to lead to silent failures?

Reply:
The previous logic requires us to add an entry for every model in `TokenizerRunExpander`.